building against mingw gcc 13.2

Getting this compile error when compiling against mingw64 gcc 13. Note the gcc81 works fine, but because of hard coding was never testing that: https://winnie.postgis.net/job/pgRouting_PGVersionEDB/11345/console

```
ordering/cuthillMckeeOrdering_driver.cpp
[ 78%] [32mBuilding C object src/ordering/CMakeFiles/ordering.dir/sloanOrdering.c.obj[0m
cd /E/jenkins/pgrouting/builddevelopw64/src/ordering && /C/ming64/mingw64/bin/cc.exe -DBOOST_ALLOW_DEPRECATED_HEADERS -DPGSQL_VERSION=18000 -DPROJECT_VERSION=\"4.0.0-alpha1\" -I/E/jenkins/pgrouting/branches/develop/include -isystem /E/jenkins/boost/rel-1_86_0w64/include/boost-1_86 -isystem /E/jenkins/postgresql/rel/pg18w64/include/server -isystem /E/jenkins/postgresql/rel/pg18w64/include/server/port/win32 -frounding-math  -g   -Wall -Wconversion -Wmissing-prototypes -W -Wunused -Wuninitialized -Wextra -Wdouble-promotion   -Wsign-conversion -MD -MT src/ordering/CMakeFiles/ordering.dir/sloanOrdering.c.obj -MF CMakeFiles/ordering.dir/sloanOrdering.c.obj.d -o CMakeFiles/ordering.dir/sloanOrdering.c.obj -c /E/jenkins/pgrouting/branches/develop/src/ordering/sloanOrdering.c
[ 78%] [32mBuilding CXX object src/ordering/CMakeFiles/ordering.dir/ordering_driver.cpp.obj[0m
cd /E/jenkins/pgrouting/builddevelopw64/src/ordering && /C/ming64/mingw64/bin/c++.exe -DBOOST_ALLOW_DEPRECATED_HEADERS -DPGSQL_VERSION=18000 -DPROJECT_VERSION=\"4.0.0-alpha1\" -I/E/jenkins/pgrouting/branches/develop/include -isystem /E/jenkins/boost/rel-1_86_0w64/include/boost-1_86 -isystem /E/jenkins/postgresql/rel/pg18w64/include/server -isystem /E/jenkins/postgresql/rel/pg18w64/include/server/port/win32 -frounding-math  -g -Wall -pedantic -W -Wunused -Wuninitialized -Wextra -Wdouble-promotion -std=c++17   -Wsign-conversion -MD -MT src/ordering/CMakeFiles/ordering.dir/ordering_driver.cpp.obj -MF CMakeFiles/ordering.dir/ordering_driver.cpp.obj.d -o CMakeFiles/ordering.dir/ordering_driver.cpp.obj -c /E/jenkins/pgrouting/branches/develop/src/ordering/ordering_driver.cpp
In file included from E:/jenkins/pgrouting/branches/develop/src/ordering/ordering_driver.cpp:33:
E:/jenkins/pgrouting/branches/develop/include/drivers/ordering_driver.hpp:45:9: error: 'int64_t' has not been declared
   45 |         int64_t**,  size_t*,
      |         ^~~~~~~
make[2]: *** [src/ordering/CMakeFiles/ordering.dir/build.make:135: src/ordering/CMakeFiles/ordering.dir/ordering_driver.cpp.obj] Error 1
make[2]: Leaving directory '/projects/pgrouting/builddevelopw64'
make[1]: *** [CMakeFiles/Makefile2:2265: src/ordering/CMakeFiles/ordering.dir/all] Error 2
make[1]: Leaving directory '/projects/pgrouting/builddevelopw64'
make: *** [Makefile:139: all] Error 2
***************************
```

--- a/include/drivers/ordering_driver.hpp
+++ b/include/drivers/ordering_driver.hpp
@@ -37,7 +37,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 #include <cstddef>
 #include <string>
-
+#include <cstdint>
 
 void
 do_ordering(

